crypto/kem/kem.c

```diff
@@ -25,39 +26,39 @@ static const uint8_t kOIDKyber1024r3[] = {0xff, 0xff, 0xff, 0xff};
 
 static const KEM built_in_kems[AWSLC_NUM_BUILT_IN_KEMS] = {
   {
-    NID_KYBER512_R3,         // kem.nid
-    kOIDKyber512r3,          // kem.oid
-    sizeof(kOIDKyber512r3),  // kem.oid_len
-    "Kyber512 Round-3",      // kem.comment
-    800,                     // kem.public_key_len
-    1632,                    // kem.secret_key_len
-    768,                     // kem.ciphertext_len
-    32,                      // kem.shared_secret_len
-    &kem_kyber512r3_method,  // kem.method
+    NID_KYBER512_R3,                // kem.nid
+    kOIDKyber512r3,                 // kem.oid
+    sizeof(kOIDKyber512r3),         // kem.oid_len
+    "Kyber512 Round-3",             // kem.comment
+    KYBER512_R3_PUBLIC_KEY_BYTES,   // kem.public_key_len
+    KYBER512_R3_PRIVATE_KEY_BYTES,  // kem.secret_key_len
```

Comment:
Nit: we are calling it `secret_key` not `private_key`.

Reply:
Fixed